src/main/java/com/delphix/masking/initializer/maskingApi/BackupDriver.java

```diff
@@ -148,6 +153,7 @@ public BackupDriver(ApplicationFlags applicationFlags) throws ApiCallException,
     public void run() {
 
         backupFileFormats();
+        backupUsers();
```

Comment:
As discussed offline, please move this into the ` if (applicationFlags.getGlobal()) {` check so that we only backup users if global objects are requested.